api/openapi/v2/core-metadata.yaml

```diff
@@ -99,8 +62,30 @@ components:
       description: "A request to add a new device profile. When making this request, the ID property should not be populated."
       type: object
       properties:
-        profile:
-          $ref: '#/components/schemas/DeviceProfile'
+        name:
```

Comment:
Can this be

```
      properties:
          $ref: '#/components/schemas/DeviceProfile'
```

So we are reusing DeviceProfile?

Reply:
Could you help take a look my comment https://github.com/edgexfoundry/go-mod-core-contracts/pull/253#discussion_r453953144?
Since AddXXXRequest and UpdateXXXRequest DTO require different field and validation tag, I'm not sure reusing DeviceProfile is correct or not.

Reply:
Got it. Thanks for looking at it.